### PR TITLE
projector.create including tests

### PR DESCRIPTION
**Pull request by r-peschke**

### CODE REVIEW

File: tests/system/action/projector/test_create.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            "projector.create",
+            {
+                "name": "test projector",
+                "meeting_id": 222,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "projector/1", {"name": "test projector", "meeting_id": 222}
+        )
```
Comment (FinnStutzenstein) [inline tests/system/action/projector/test_create.py:21]: Can you check, that the defaults are correct?

File: tests/system/action/projector/test_create.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    def test_create_all_fields(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV"},
+            }
+        )
+        data = {
+            "name": "Test",
+            "meeting_id": 222,
+            "width": 100,
```
Comment (FinnStutzenstein) [inline tests/system/action/projector/test_create.py:32]: As I see this test: `width`, `aspect_ratio_numerator` and `aspect_ratio_denominator` must be `>=1`. Is this possible to do in the models.yml? Please add this somewhere (the update also needs this info).

**Review by FinnStutzenstein — APPROVED** [review]
Looks good, minor changes needed.

File: tests/system/action/projector/test_create.py (hunk 1 above)
Comment (r-peschke) [reply tests/system/action/projector/test_create.py:21]: Done

File: tests/system/action/projector/test_create.py (hunk 2 above)
Comment (r-peschke) [reply tests/system/action/projector/test_create.py:32]: done, but the https://github.com/OpenSlides/OpenSlides/pull/5914 with the changed models.yml is necessary, too

File: tests/system/base.py [hunk 3]
```diff
@@ -157,3 +157,12 @@ def assert_model_not_exists(self, fqid: str) -> None:
...
         model = self.get_model(fqid)
         self.assertTrue(model.get("meta_deleted"))
+
+    def assert_defaults(self, model: Type[Model], instance: Dict[str, Any]) -> None:
+        for field in model().get_fields():
+            if hasattr(field, "default") and field.default is not None:
+                self.assertEqual(
+                    field.default,
+                    instance.get(field.own_field_name),
+                    f"Field {field.own_field_name}: Value {instance.get(field.own_field_name, 'NONE')} is not equal Default Value {field.default}.",
```
Comment (FinnStutzenstein) [inline tests/system/base.py:167]:
```suggestion
                    f"Field {field.own_field_name}: Value {instance.get(field.own_field_name, 'None')} is not equal default value {field.default}.",
```

**Review by FinnStutzenstein — APPROVED** [review]

File: tests/system/base.py (hunk 3 above)
Comment (r-peschke) [reply tests/system/base.py]: done

**Pull request merged**